lib/node_modules/@stdlib/stats/base/dists/rayleigh/cdf/benchmark/benchmark.js

```diff
@@ -42,8 +42,8 @@ bench( pkg, function benchmark( b ) {
 	x = new Float64Array( len );
 	sigma = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		x[ i ] = ( randu() * 100.0 ) - 100.0;
-		sigma[ i ] = ( randu() * 20.0 ) + EPS;
+		x[ i ] = uniform( -100.0, 100.0 );
```

Comment:
Changing the range to `[-100, 100]` makes sense, not sure why we were only generating negative `x` previously.